lib/matplotlib/tests/test_getattr.py

```diff
@@ -4,14 +4,20 @@
 import matplotlib
 import pytest
 
-# Get the names of all matplotlib submodules, except for the unit tests.
-module_names = [m.name for m in walk_packages(path=matplotlib.__path__,
-                                              prefix=f'{matplotlib.__name__}.')
-                if not m.name.startswith(__package__)]
+# Get the names of all matplotlib submodules,
+# except for the unit tests and private modules.
+module_names = [
+    m.name
+    for m in walk_packages(
+        path=matplotlib.__path__, prefix=f"{matplotlib.__name__}."
+    )
+    if not m.name.startswith(__package__)
+    and not any(x.startswith("_") for x in m.name.split("."))
+]
 
 
-@pytest.mark.parametrize('module_name', module_names)
-@pytest.mark.filterwarnings('ignore::DeprecationWarning')
+@pytest.mark.parametrize("module_name", module_names)
```

Comment:
Is this and the following from a code formatter? My understanding is that we don't want code churn like this. If we want to introduce more strict formatting, we need to decide this centrally and introduce this in a coordinated way.

Reply:
Guilty... I did apply `black` (only to this file) when flake8 started complaining about the formatting of the module name filter

I can revert the quote changes if we want.